test/fixtures/rules/std_rule_cases/L028.yml

```diff
@@ -4,6 +4,22 @@ rule: L028
 test_fail_single_table_mixed_qualification_of_references:
   fail_str: SELECT my_tbl.bar, baz FROM my_tbl
 
+test_fail_mixed_qualification_fix_to_unqualified:
+  fail_str: SELECT my_tbl.bar, baz FROM my_tbl
+  fix_str: SELECT bar, baz FROM my_tbl
+  configs:
+    rules:
+      L028:
+        fix_inconsistent_to: unqualified
+
+test_fail_mixed_qualification_fix_to_qualified:
+  fail_str: SELECT my_tbl.bar, baz FROM "my_tbl"
+  fix_str: SELECT my_tbl.bar, "my_tbl".baz FROM "my_tbl"
```

Comment:
This is a bit weird as different references for same table. I can see why it's done it (only needed to fix the second one) but should we do a simpler test without quote to avoid this confusing people? Or is it specifically the quotes you want to test her?

Reply:
As per the description it is a bit weird, but its already like this (at least in terms of passing). As I further mentioned How does one validate that they are building the right tree, or how does one leave this up to the next/another Rule to Fix (which seems logical).

I popped in the test, because it is like this and the test at least flag that this is the current bahaviour